### PR TITLE
[IMP] mail: adapt message tests to use real parent (discuss/chatter)

### PR DESCRIPTION
**Pull request by seb-odoo**

### CODE REVIEW

File: addons/mail/static/src/models/timer.js [hunk 1]
```diff
@@ -83,6 +91,10 @@ registerModel({
             readonly: true,
             required: true,
         }),
+        messagingAsFetchImStatusTimerOwner: one('Messaging', {
```
Comment (alexkuhn) [inline addons/mail/static/src/models/timer.js:94]: I prefer `messagingOwnerAs...`

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Comment by seb-odoo** [comment]
Fixed runbot

@robodoo r+

**Comment by seb-odoo** [comment]
Well actually enterprise still needs adaptations as well...
@robodoo r-

**Comment by seb-odoo** [comment]
@robodoo r+

**Pull request closed without merging**